frontend/src/app/revisions.tsx

```diff
@@ -293,15 +306,15 @@ class Revisions extends Component<IProps> {
                                 />
                             </div>
                         )}
-                        {isDup && (
+                        {/* {this.state.isDup && (
```

Comment:
If we are not using this code, we should just delete it instead of commenting it out.

Reply:
removed